provisioning/miniprov/hare_mp/templates/hare.config.conf.tmpl.json

```diff
@@ -6,11 +6,9 @@
       },
       "storage_set": [
         {
-          "durability": {
-            "data": "1",
-            "parity": "0",
-            "spare": "0"
-          },
+          "data": "1",
```

Comment:
@SwapnilGaonkar7 Please update create-template.sh script accordingly.

Reply:
@knekrasov You mean this provisioning/miniprov/create-template.sh?

Reply:
@SwapnilGaonkar7 Yes, that one.